plugins/CoreHome/angularjs/reporting-menu/reportingmenu-model.js

```diff
@@ -161,5 +163,38 @@
                 return model.menu;
             });
         }
+
+        function fetchVisits(date, period, callback) {
+            var ajaxRequest = new ajaxHelper();
+
+            ajaxRequest.addParams({
+                module: 'API',
+                method: 'VisitsSummary.getVisits',
+                format: 'json',
+                date: date,
+                period: period,
```

Comment:
if it should work with segments you may need to add the segment parameter to this and the query below, to ensure it fetches the segmented data.

Reply:
interestingly it is there. but I'll double check it, and even it's there I'll add it.

Reply:
I checked the `piwikApi` and it adds all the parameters by default (idSite, period, segment, date...)